apiserver/facades/client/charms/service.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
```

Comment:
Please rename this file and other similar service.go files to interface.go to be consistent with the repo.

Reply:
`service.go` is the convention we've been using in every other facade for a file containing all the service interfaces used by the facade.